src/metabase/driver/sql_jdbc/sync/describe_table.clj

```diff
@@ -175,24 +175,40 @@
    (describe-table-fields-xf driver table)
    (fields-metadata driver conn table db-name-or-nil)))
 
+(defmulti get-table-pks
+  "Returns a set of primary keys for `table` using a JDBC DatabaseMetaData from JDBC Connection `conn`.
+  Note: If db-name, schema, and table-name are not passed, this may return _all_ pks that the metadata's connection can access."
+  {:added    "0.45.0"
+   :arglists '([driver ^Connection conn db-name-or-nil table])}
+  driver/dispatch-on-initialized-driver
+  :hierarchy #'driver/hierarchy)
```

Comment:
Don't forget to document driver API changes in `driver-changelog.md`

https://github.com/metabase/metabase/blob/master/docs/developers-guide/driver-changelog.md